extensions/mssql/src/constants/locConstants.ts

```diff
@@ -1408,7 +1408,7 @@ export class PublishProject {
         return l10n.t("Failed to fetch Docker container tags: {0}", errorMessage);
     };
     public static ProfileLoadedConnectionFailed = l10n.t(
```

Comment:
We shouldn't have two identical error messages as separate entries here.  It's okay for Schema Compare to call a PublishProject string or vice-versa.